tools/testdebug/cmd/load_etcd.go

```diff
@@ -22,7 +22,7 @@ import (
 	authorizationclient "github.com/openshift/origin/pkg/authorization/generated/internalclientset"
 	"github.com/openshift/origin/pkg/cmd/flagtypes"
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
-	configapi "github.com/openshift/origin/pkg/cmd/server/config"
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
```

Comment:
How is this working before? may be this code was not tested as it is under tools/testdebug?